managed/services/supervisord/logs.go

```diff
@@ -217,38 +217,45 @@ func (l *Logs) files(ctx context.Context, pprofConfig *PprofConfig) []fileConten
 
 	// add pprof
 	if pprofConfig != nil {
+		filesSync := &sync.Mutex{}
```

Comment:
This one is not related to PR's goal. 
I found race problem (with appending to files array) and decided to fix it.